GVFS/GVFS.Common/InternalVerbParameters.cs

```diff
@@ -4,16 +4,22 @@ namespace GVFS.Common
 {
     public class InternalVerbParameters
     {
-        public InternalVerbParameters(string serviceName, bool startedByService, string maintenanceJob)
+        public InternalVerbParameters(
+            string serviceName = null,
+            bool startedByService = true, 
+            string maintenanceJob = null, 
+            string batchSize = null)

```

Comment:
Can you explain why we want to pass batchsize as a parameter.  This is only for testing correct?  I've tried to not add extra parameters here where I can.

Reply:
Is this because the default data isn't big enough to be repacked?

Reply:
correct.

Reply:
It looks like we always use DefaultBatchSize = "2g"; in the tests.  Can we instead of passing in another param, use 2g as a default in DehydrateVerb?

InternalVerbParameters have to be passed on  the command line since it's serizlied json. It's tough to write the more we have.  Perhaps this is an argument for not using serialized json and going back to comma seperated field/value pairs which I originally did and was asked to change.